lib/astutils.cpp

```diff
@@ -536,7 +536,7 @@ const Token* getParentMember(const Token * tok)
         return parent->astOperand1();
     }
     const Token * gparent = parent->astParent();
-    if (!Token::simpleMatch(gparent, ".") || gparent->astOperand2() != parent)
```

Comment:
is this safe? if gparent is null then simpleMatch would return false

Reply:
Yes are right. I thought the same but somehow did commit it. Will revert.

Reply:
done.